Describe env specific ActiveStorage config in the AS guide [skip ci] (#46599)
rails/rails: guides/source/active_storage_overview.md

config.active_storage.service = :test
```

Continue reading for more information on the built-in service adapters (e.g.

> `Disk` and `S3`) and the configuration they require.

Author: Not really related but moving this section to the bottom follows the flow better.

[fixtures]: testing.html#the-low-down-on-fixtures
[`ActiveStorage::FixtureSet`]: https://api.rubyonrails.org/classes/ActiveStorage/FixtureSet.html

### Configuring services

> You can add `config/storage/test.yml` to configure services only when testing.

Reviewer: ```suggestion
You can add `config/storage/test.yml` to configure services to be used in test environment
```

This is useful when the `service` option is used.

```ruby
class User < ApplicationRecord
  has_one_attached :avatar, service: :s3
end
```

> Without `config/storage/test.yml` the `s3` service configured in

Reviewer: ```suggestion
Without `config/storage/test.yml`, the `s3` service configured in
```

> `config/storage.yml` is used that would end up sending actual requests to S3.

Author: @Buzzkill-McSquare Thank you!

> `config/storage.yml` is used - even when running tests, the default configuration would be used and files would be uploaded to the service provider configured in `config/storage.yml`.

Reviewer: ```suggestion
`config/storage.yml` is used - even when running tests. 
The default configuration would be used and files would be uploaded to the service provider configured in `config/storage.yml`.
```

[fixtures]: testing.html#the-low-down-on-fixtures
[`ActiveStorage::FixtureSet`]: https://api.rubyonrails.org/classes/ActiveStorage/FixtureSet.html

### Configuring services

You can add `config/storage/test.yml` to configure services to be used in test environment
This is useful when the `service` option is used.

```ruby
class User < ApplicationRecord
  has_one_attached :avatar, service: :s3
end
```

> Without `config/storage/test.yml`, the `s3` service configured in
> `config/storage.yml` is used - even when running tests. 
> The default configuration would be used and files would be uploaded to the service provider configured in `config/storage.yml`.
> In this case, you can add `config/storage/test.yml` and use Disk service for
> `s3` service to prevent sending requests.

Reviewer 2: ```suggestion
Without `config/storage/test.yml`, the `s3` service configured in `config/storage.yml` is used - even when running tests.

The default configuration would be used and files would be uploaded to the service provider configured in `config/storage.yml`.

In this case, you can add `config/storage/test.yml` and use Disk service for `s3` service to prevent sending requests.
```

Reviewer 2: This should either be one paragraph, ideally wrapped around 80 columns, or 3 separate ones -- whichever fits best.
